holidays/groups/international.py

```diff
@@ -11,14 +11,25 @@
 
 from datetime import date
 
-from holidays.constants import JAN, MAR, MAY, OCT, NOV, DEC
+from holidays.constants import JAN, MAR, MAY, JUN, OCT, NOV, DEC
 
 
 class InternationalHolidays:
     """
     International holidays.
     """
 
+    def _add_childrens_day_jun1(self, name):
```

Comment:
I believe we can combine these 2 holidays within one method with an optional parameter, for example`def _add_childrens_day(self, name, month="JUN"):` or similar. Adding _jun1 suffix to the name seems inconsistent with existing methods. It may be even better to keep it simple and not add the November related stuff at all until we need it.

Reply:
Fine for me IMO